core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -2117,4 +2117,13 @@ package object config {
       // batch of block will be loaded in memory with memory mapping, which has higher overhead
       // with small MB sized chunk of data.
       .createWithDefaultString("3m")
+
+  private[spark] val MARK_FILE_LOST_ON_EXECUTOR_LOST =
+    ConfigBuilder("spark.shuffle.markFileLostOnExecutorLost")
+      .doc("Mark shuffle files as lost when an executor is lost. If you are using a remote " +
+       "shuffle service such that shuffle files are not stored on the executor, consider " +
```

Comment:
Well. In this context, `remote shuffle service` is not `external shuffle service` in Apache Spark because we are ignoring `env.blockManager.externalShuffleServiceEnabled` here. We have better be clear on this.